### PR TITLE
V1.3.0 release

### PR DESCRIPTION
**Pull request by angrybayblade**
## Release summary

Version number: 1.3.0

## Release details

AEA:
- Adds support to scaffold packages outside an AEA project
- Adds support for IPFS package hashing and IPFS based registry.
- Allows contracts to depend on other contracts.

Plugins:
- Adds support for EIP1559 based gas estimation strategy.

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
 
+## 1.3.0 (2021-12-31)
+
+AEA:
+- Adds support to scaffold packages outside an AEA project
+- Adds support for IPFS package hashing and IPFS based registry.
+- Allows contracts to depend on other contracts.
+
+Plugins:
+- Adds support for EIP1559 based gas estimation strategy.
```
Comment (angrybayblade) [inline HISTORY.md:11]: @DavidMinarsch Am I missing anything here ?

Comment (DavidMinarsch) [reply HISTORY.md:11]: You could add a docs section and reference the docs updates:
- Applies new styling
- Simplifies documentation and updates with latest features

**Review by DavidMinarsch — COMMENTED** [review]
What about plugin version update?

**Review by DavidMinarsch — APPROVED** [review]
LGTM

**Pull request merged**